mne/decoding/time_gen.py

```diff
@@ -123,8 +123,13 @@ def fit(self, epochs, y=None):
         If X is a dense array, then the other methods will not support sparse
         matrices as input.
         """
-        from sklearn.base import clone
-        from sklearn.cross_validation import check_cv, StratifiedKFold
+        from sklearn.base import clone, is_classifier
+        try:
+            from sklearn.model_selection import (check_cv, StratifiedKFold,
+                                                 KFold)
+        except Exception:  # XXX support sklearn < 0.18
+            from sklearn.cross_validation import (check_cv, StratifiedKFold,
+                                                  KFold)
```

Comment:
we really should try to factor out `cv` object to avoid breaking this module whenever `scikit-learn` updates.